config/config.go

```diff
@@ -37,21 +36,21 @@ func init() {
 	Config = GetConfig()
 }
 
-func GetConfig() *ConfigWrapper {
+func GetConfig() *configWrapper {
```

Comment:
Returning variables of private types from public functions is not a recommended practice

Reply:
can you link me to where this is not recommended? i see this in plenty of places and it makes sense to me

Reply:
I learned this from golint and [their reasoning](seems compelling), namely:

> can't put a *sample field in their own struct

Even in my not-so-long practice I distinctly remember being annoyed by this more than a few of times.